specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/preview/2022-08-01-preview/nodetype.json

```diff
@@ -899,6 +910,10 @@
         "evictionPolicy": {
           "$ref": "#/definitions/EvictionPolicyType",
           "description": "Specifies the eviction policy for virtual machines in a SPOT node type. Default is Delete."
+        },
+        "vmImageResourceId": {
+          "type": "string",
```

Comment:
you might want to add http://azure.github.io/autorest/extensions/#x-ms-arm-id-details

same for other one that looks like ARM resource id.

Reply:
Thanks, added. Will it be considered a breaking change if we add that for parameters introduced in older apis?

Reply:
I am not so sure about existing properties.

Currently there are valuable for track2 .NET SDK. Loop in @ArthurMa1978 for .NET

Reply:
@a-santamaria can you provide the list of `arm-id` property / parameter?